### ISSUE TITLE
Script error.

### ISSUE THREAD

**Issue by andrewlyons** [EDCD/coriolis#39]
URL was:
https://coriolis.edcd.io/outfit/federal_gunship?code=3pktkFflndusmf47l272727271010040402004f5nfr016b.Iw18aQ%3D%3D.CwBhGYzBGXYJnlKlhA%3D%3D.

Script error.

Create an issue on Github if this keeps happening. Add these details:
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.98 Safari/537.36
Path: /outfit/federal_gunship?code=3pktkFflndusmf47l272727271010040402004f5nfr016b.Iw18aQ%3D%3D.CwBhGYzBGXYJnlKlhA%3D%3D.
Error: Unknown
Details:
scriptUrl: 
line: 0
col: 0
error: null
2.2.3 - 2016-11-24

**Comment by cmdrmcdonald** [comment]
This URL works fine when I try it.  Are you running ghostery or similar?

**Comment by Mundo54** [comment]
I was getting the script error too so I clicked on the link to come here and I saw the Ghostery comment.

Trusting the site in Ghostery solved the problem for me. Thanks for suggesting that, it never dawned on me that Ghostery could be the problem. I had assumed it was a recent Firefox update that was the issue.

On a related side note, while shortcuts and links to builds would fail, I could still use the BUILDS option from the menu to access stored builds on the site.
